backend/btrixcloud/basecrawls.py

```diff
@@ -166,6 +168,7 @@ async def delete_crawls(
             size += await self._delete_crawl_files(crawl, org)
             if crawl.get("cid"):
                 cids_to_update.add(crawl.get("cid"))
+            await inc_org_bytes_stored(self.orgs_db, org.id, -size)
```

Comment:
Should we move this to be after the deletion operation, just in case?

Reply:
Yeah good call! That way it only runs once too